src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -1103,7 +1103,9 @@ public static class MenuNames
          public static readonly string NewFormulation = "Add &Formulation...";
          public static readonly string NewEvent = "Add &Event...";
          public static readonly string AddObservedData = "Add &Observed Data...";
-         public static readonly string AddObservedDataFor = "Add Observed Data for";
+         public static readonly string AddAmountObservedData = "Add &Amount Observed Data...";
+         public static readonly string AddConcentrationObservedDataFor = "Add Observed Data for";
+         public static readonly string AddAmountObservedDataFor = "Add Amount Observed Data for";
```

Comment:
Acutally I think the best would be to have
Add Amount Data
Add Concentration Data
Add Fraction Data 

Forget about observer all together

Reply:
s. above